src/openclsim/core/locatable.py

```diff
@@ -1,17 +1,17 @@
-"""Component to locate the simulation objecs."""
+"""Component to locate the simulation objects."""
 import pyproj
 import shapely.geometry
 
 
 class Locatable:
     """
     Something with a geometry (geojson format).
+    Can be a point as well as a polygon
 
     Parameters
     ----------
-    lat : degrees
-        can be a point as well as a polygon
-    lon : degrees
+    lat : degrees (wgs84)
```

Comment:
Are these used? The code uses `geometry`.

Reply:
indeed. I did just cosmetics, so this should be geometry